src/jdk.hotspot.agent/share/classes/sun/jvm/hotspot/runtime/Threads.java

```diff
@@ -163,8 +164,8 @@ public Threads() {
 
     /** NOTE: this returns objects of type JavaThread, CompilerThread,
       JvmtiAgentThread, NotificationThread, MonitorDeflationThread,
-      StringDedupThread, AttachListenerThread and ServiceThread.
-      The latter seven subclasses of the former. Most operations
+      StringDedupThread, AttachListenerThread, DeoptimizeObjectsALotThread and
+      ServiceThread. The latter seven subclasses of the former. Most operations
```

Comment:
The latter eight are subclasses of the former? 8-) Or just "JavaThread, or its subclasses..."

Reply:
Yes, it should be updated to eight, but I think what would be better is "returns objects of type JavaThraed or one of its subclasses..."

Reply:
Ok, it's fixed now. I stuck with a somewhat minimal update mostly just to resolve the incorrect count. This comment may have further cleanup as part of [JDK-8348347](https://bugs.openjdk.org/browse/JDK-8348347).